tests/shortcodesTest.php

```diff
@@ -428,8 +383,8 @@ public function testDoShortcodeContent() {
 			$this->fail( 'Failing this test because the country is not US or the region is not UT.' );
 		}
 		$this->assertEquals(
-			Shortcodes\do_shortcode_content( $shortcode, $content ),
 			'',
+			Shortcodes\do_shortcode_content( $shortcode, $content ),
```

Comment:
I see several diffs where the order of the function calls are changed.  Can you explain this a bit for me?

Reply:
Yeah, I found through testing this that the _expected_ result should be the first parameter when doing `assertEquals` checks. This became evident through messages like this:

```
[geoip-content continent="NA" not_country="US"] does not match expected output.
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-''
+'This should be hidden from US visitors.'
```

In the case above the _Expected_ is being shown as the _Actual_ result which is inaccurate.